.github/workflows/docs.yml

```diff
@@ -12,33 +12,6 @@ on:
         - master
 
 jobs:
-  broken-links:
```

Comment:
@jajapuramshivasai Changes from other PRs are showing up here.

Reply:
How are you doing the rebases/merges? It likely has to do with that.